src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/XmlDataListSource.cs

```diff
@@ -154,5 +174,35 @@ public XmlNotesConfigurationField()
 </div>", true)
             { }
         }
+
+
+        /// <summary>
+        /// Returns an instance of XmlNamespaceManager with a specified XmlDocument.NameTable.  
+        /// Returns null if no namespace is defined.
+        /// </summary>
+        /// <param name="doc">The XmlDocument</param>
+        /// <returns>XmlNamespaceManager if there is at least one namespace, null if there is no namespace defined.</returns>
+        private XmlNamespaceManager CreateNamespaceManager(XmlDocument doc)
```

Comment:
@renetruelsen Just checking, was this code taken from StackOverflow answer? https://stackoverflow.com/a/26786080/12787
If so, we'll need to add in licensing info. All code from StackOverflow is licensed under CC BY-SA 4.0 (requires attribution). 👍 

I did similar on the "bytes.js": https://github.com/leekelleher/umbraco-contentment/blob/master/src/Umbraco.Community.Contentment/DataEditors/Bytes/bytes.js#L1-L4

Reply:
Thanks for info. I didn't know that, but I'll remember that for future references. 
This specific code has been removed though. Basically, from what I read, namespaces are generally a bad match with generic XML from different sources. That's why I prefer the new approach. :)